app/controllers/ems_common.rb

```diff
@@ -834,6 +837,8 @@ def get_form_vars
     @edit[:new][:bearer_token] = params[:bearer_token] if params[:bearer_token]
     @edit[:new][:bearer_verify] = params[:bearer_verify] if params[:bearer_verify]
 
+    @edit[:new][:service_account] = params[:service_account] if params[:service_account]
+
```

Comment:
```params``` data from the Cloud Provider is directly saved to the ```ems``` record in  ```set_ems_record_vars``` in ```ems_cloud_controller```
Looks like you are already saving it there -
https://github.com/lwander/manageiq/blob/google_fog_auth/app/controllers/ems_cloud_controller.rb#L214